### PR TITLE
FORMS-2980: default validation - synced fieldType email with af2-web-runtime

### PR DESCRIPTION
**Pull request by egmox**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/emailinput/v1/emailinput/_cq_template.xml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
     jcr:primaryType="nt:unstructured"
-    jcr:title="Email Input"
-    fieldType="text-input"/>
+    jcr:title="Email Input"/>
```
Comment (nit23uec) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/emailinput/v1/emailinput/_cq_template.xml:4]: just `Email` in the title is enough.

**Comment by barshat7** [comment]
We will need to update the fieldType in archetype also https://github.com/adobe/aem-project-archetype/blob/ff01ddb5ffc86b597e46d7daa69d75d22775187c/src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/emailinput/_cq_template.xml#L5
@egmox  Please raise a Jira for this.

**Review by nit23uec — APPROVED** [review]

File: ui.frontend/package.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
     "webpack-merge": "^5.8.0"
   },
   "dependencies": {
-    "@aemforms/af-core": "^0.22.37",
+    "@aemforms/af-core": "0.22.38",
```
Comment (rismehta) [inline ui.frontend/package.json:26]: You need to add ^ here

File: ui.tests/test-module/specs/emailinput/emailinput.authoring.spec.js [hunk 3]
```diff
@@ -74,6 +74,26 @@ describe('Page - Authoring', function () {
             testEmailInputBehaviour(emailInputEditPathSelector, emailInputDrop);
         });
     });
+
+    it('Invalid email input should show error message', { retries: 3 }, function () {
+      dropEmailInputInContainer();
+      cy.get('[data-layer="Preview"]').click();
+      getPreviewIframeBody().find('input.cmp-adaptiveform-emailinput__widget')
+          .should('be.visible').type('invalidEmail', {force: true}).blur({force: true});
+      getPreviewIframeBody().find('.cmp-adaptiveform-emailinput__errormessage').should('exist');
```
Comment (rismehta) [inline ui.tests/test-module/specs/emailinput/emailinput.authoring.spec.js:83]: This test needs to be written in emailinput.runtime.spec.js. Let's not access iframe details inside authoring

**Review by rismehta — CHANGES_REQUESTED** [review]
-1 for now

**Comment by egmox** [comment]
@rismehta resolved the comments

**Review by rismehta — APPROVED** [review]
LGTM

**Pull request merged**